src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -283,6 +288,40 @@ static boolean hasOcr(CaseDetails theCase) {
         return valid(classification);
     }
 
+    public static Validation<String, Classification> hasJourneyClassificationForAttachToCase(CaseDetails theCase) {
+        Optional<String> classificationOption = getJourneyClassification(theCase);
+
+        return classificationOption
+            .map(classification -> Try.of(() -> Classification.valueOf(classification)))
+            .map(Try::toValidation)
+            .map(validation -> validation
+                .mapError(throwable -> "Invalid journeyClassification. Error: " + throwable.getMessage())
+                .flatMap(classification -> validateClassificationForAttachToCase(classification, theCase))
+            )
+            .orElse(invalid("Missing journeyClassification"));
+    }
+
+    private static Validation<String, Classification> validateClassificationForAttachToCase(
+        Classification classification,
+        CaseDetails theCase
+    ) {
+        if (!VALID_CLASSIFICATIONS_FOR_ATTACH_TO_CASE.contains(classification)) {
+            return invalid(format(
+                "The current journey classification %s is not allowed for attaching to case",
+                classification
+            ));
+        }
+
+        if (SUPPLEMENTARY_EVIDENCE_WITH_OCR.equals(classification) && !hasOcr(theCase)) {
+            return invalid(format(
+                "The current journey classification %s is not allowed without OCR",
+                classification
+            ));
+        }
```

Comment:
I'd move OCR presence validation outside classification, because it's misleading as a part of classification validation.

Also, worth adding `data` to the end of the message, because while we internally think about `OCR` as `OCR data`, it may not be obvious to the caseworker. OCR stands for Optical Character Recognition.